client_v25.go

```diff
@@ -258,7 +258,7 @@ func (d *Dgraph) Close() {
 
 // signInUser logs the user in using the provided username and password.
 func (d *Dgraph) signInUser(ctx context.Context, username, password string) error {
-	if d.useV24 {
+	if d.useV1 {
```

Comment:
Why is signInUser and ping privately scoped? Are they not available to the dgo end user?

Reply:
We don't need signInUser like we had discussed in the call together because Open, NewClient and NewClientWithRoundRobin all accepts ACL credentials as a ClientOption.

Reply:
There is no need to expose ping, we can if and when needed.

Reply:
Cool.... I've always wanted a Ping in all clients.